pkgs/by-name/li/libid3tag/package.nix

```diff
@@ -23,6 +23,12 @@ stdenv.mkDerivation rec {
     ./CVE-2017-11550-and-CVE-2017-11551.patch
   ];
 
+  # this can be removed when upstream autoreconfs itself
+  # since it's broken for FreeBSD and autoreconfHook doesn't work
```

Comment:
Calling plain autoreconf also doesn't work?

Reply:
It doesn't, no.

Probably the easiest way to do this "right" is to just do an upgrade. Arch Linux has a newer version and lists https://codeberg.org/tenacityteam/libid3tag as upstream.

Reply:
Debain, too https://salsa.debian.org/multimedia-team/libid3tag

Reply:
Yep, sounds good.

Reply:
#387670